services/transactionsService.ts

```diff
@@ -7,14 +7,23 @@ import { fetchPaybuttonAddressBySubstring } from 'services/paybuttonAddressesSer
 async function getReceivedAmount (transaction: BCHTransaction.AsObject, receivingAddress: string): Promise<Prisma.Decimal> {
   let totalOutput = 0
   transaction.outputsList.forEach((output) => {
-    if (output.address === receivingAddress) {
+    if (receivingAddress.includes(output.address)) {
       totalOutput += output.value
     }
   })
-  return new Prisma.Decimal(totalOutput).dividedBy(10e8)
+  return new Prisma.Decimal(totalOutput).dividedBy(1e8)
 }
 
-export async function saveTransaction (transaction: BCHTransaction.AsObject, receivingAddress: string): Promise<Transaction | undefined> {
+export async function fetchAddressTransactions (address: string): Promise<Transaction[]> {
+  const paybuttonAddress = await fetchPaybuttonAddressBySubstring(address)
+  return paybuttonAddress.receivedTransactions.sort(function (a, b) {
```

Comment:
I'm surprised we don't have it already - lets import lodash and just use `_.sortBy()` or `_.orderyBy()` since I'm sure this is not the first or last time we'll want to sort an array of objects in this project

Reply:
Addressed.